### ISSUE TITLE
Clicking on image not do bash/href action

### ISSUE THREAD

**Issue by trmdi** [lipido/kargos#5]
```bash
echo "| bash='$base_dir/ibus_switch.sh' onclick=bash imageURL=$icon size=5"
```
I want to create a non-title instance of kargos like this one (without a title before `|`). But in this case clicking on the image cause nothing to happen.

Please add support for this.

**Comment by lipido** [comment]
Fixed in c02725c4e4c3ea096cbe8b7ee3ec08220a0920a0
Please, test.

**Comment by trmdi** [comment]
Well, the bug has been fixed very quickly. Thank you.
